mobile/src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java

```diff
@@ -397,6 +400,8 @@ private void tearDownState(Bundle state) {
     public void onDestroyView() {
         super.onDestroyView();
 
+        SharedVariables.isAlertsFragmentAvailable = false;
```

Comment:
If you are stuck because of static. you can access element of TriggersFragment with `getTriggersFragment().isTriggersFragmentAvailable`